service/frontend/adminHandler.go

```diff
@@ -1727,11 +1727,32 @@ func (adh *adminHandlerImpl) ListDynamicConfig(ctx context.Context, request *typ
 }
 
 func (adh *adminHandlerImpl) GetGlobalIsolationGroups(ctx context.Context, request *types.GetGlobalIsolationGroupsRequest) (_ *types.GetGlobalIsolationGroupsResponse, retError error) {
-	panic("not implemented")
+	defer func() { log.CapturePanic(recover(), adh.GetLogger(), &retError) }()
+	scope, sw := adh.startRequestProfile(ctx, metrics.GetGlobalIsolationGroups)
+	defer sw.Stop()
+
+	if request == nil {
+		return nil, adh.error(errRequestNotSet, scope)
+	}
+	ig := adh.Resource.GetIsolationGroupState()
+	if ig == nil {
+		return nil, nil
```

Comment:
I think it's better to return `&types.GetGlobalIsolationGroupsResponse{}, nil`

Reply:
good call